privacy_policy/en/privacy-notice-website.md

```diff
@@ -0,0 +1,65 @@
+# Adguard website & Cookies
+*August 03, 2017*
+> **Keynote:** Adguard does not share/sell any of your personal data. We are proud to say that we are one of the fighters for users privacy and we are strongly committed to this principle and to being as transparent as possible.
+
+This privacy notice applies to *adguard.com* and all the Adguard websites located on the subdomains, and describes in detail what information may be collected about you when you visit them.
+
+## Cookies
+When you visit our website, our server may place a small data file called a «cookie» (sometimes called «web cookie» or «browser cookie») on your hard drive. Most commercial websites use cookies. By using our site you agree to the use of cookies as described in this privacy notice... 
+(LEARN MORE)
+The data we acquire from cookies is processed by various analytics tools and may be used to: 
+* Determine your geographic location (via your IP address and automated geolocation techniques). Geolocation data helps us to deliver content in the correct language, offer region- or country-specific products, and identify your local currency in the event of a purchase.
+Acquire basic information about the computer, tablet, or mobile phone that you use to visit us. This helps us to offer you the right product.
+* Collect anonymous information in order to improve the accuracy of our services. We can also use cookies to collect anonymous information regarding downloads of our products from download site(s); this information is used only for statistical purposes, and *no personal information is collected or stored*. 
+* Help you log in by remembering your user credentials (depending on which options you select in your profile settings). We may also use cookies to record user registration information. Cookies can help us identify your preferences, including language and currency, for future use. Cookies usually expire within 30-60 days.
+
+Important: If you wish, you can always [disable cookies](http://www.wikihow.com/Disable-Cookies) on your web browsing software. But please note that certain features of our products and services may not function properly without the aid of cookies.
+(/LEARN MORE)
+
+## Interacting with Adguard websites
+### When you visit our website 
+We may use cookies, third-party web analytics, GIFs, device information, and IP addresses for functionality and to better understand the use of our site by our visitors. [LEARN MORE] 
+
+**Analytics & Metrics**: We currently use several website analytics tools. These tools help us understand the use of our site by our visitors by collecting information. 
+
+Piwik (piwik.adguard.com) which we consider the best option from the privacy point of view as data is not shared with third parties (service providers) and we manage it ourselves. In the nearest future we will switch over to this service only. 
```

Comment:
Let's replace "In the nearest future" with "Eventually". We're not very good with estimating due dates you know :)